src/runner/htmlRunner.ts

```diff
@@ -1,12 +1,18 @@
 import { IOptions, Result } from '..'
 import { Context } from '../types'
 
-export const ERROR_HANDLING_SCRIPT = `<script>
+const ERROR_HANDLING_SCRIPT_TEMPLATE = `<script>
   window.onerror = (msg, url, lineNum) => {
-    window.parent.postMessage("Line " + Math.max(lineNum - 5, 0) + ": " + msg, "*");
+    window.parent.postMessage("Line " + Math.max(lineNum - %d, 0) + ": " + msg, "*");
   };
 </script>\n`
 
+const errorScriptLines = ERROR_HANDLING_SCRIPT_TEMPLATE.split('\n').length - 1
+export const htmlErrorScript = ERROR_HANDLING_SCRIPT_TEMPLATE.replace(
```

Comment:
Can we standardise the names? Why does `ERROR_HANDLING_SCRIPT_TEMPLATE` produce an `htmlErrorScript`?